hathor/event/event_manager.py

```diff
@@ -0,0 +1,55 @@
+import time
+
+from structlog import get_logger
+
+from hathor.event.base_event import BaseEvent
+from hathor.event.storage import EventStorage
+from hathor.pubsub import EventArguments, HathorEvents, PubSubManager
+
+logger = get_logger()
+
+
+class EventManager:
+    def __init__(self, event_storage: EventStorage, peer_id: str):
+        self.event_storage = event_storage
+        last_event = event_storage.get_last_event()
+        self.next_event_index = 1 if last_event is None else last_event.id+1
```

Comment:
It's weird that our linter didn't get it. I feel we need to add some space here: `last_event.id + 1`.

Reply:
Yes. I fixed this in a later commit, I'm pulling those changes, since I think they make sense.

Reply:
Fixed.

Reply:
Thanks!